src/osd/osd_types.cc

```diff
@@ -4143,11 +4143,12 @@ void pg_log_t::filter_log(spg_t import_pgid, const OSDMap &curmap,
 
 void pg_log_t::encode(bufferlist& bl) const
 {
-  ENCODE_START(7, 3, bl);
+  ENCODE_START(8, 3, bl);
   encode(head, bl);
   encode(tail, bl);
   encode(log, bl);
   encode(can_rollback_to, bl);
+  encode(rollback_invalidate, bl);
```

Comment:
when adding fields, add them to teh end of the encoding so that older version of the code can still find the fields the expect